<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Управление</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Схема:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>unknown</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Хост:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Порт:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>unknown</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>unknown</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Комнаты</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Удалить комнату чата</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>Создать комнату</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="comboBox_2"/>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QGroupBox" name="groupBox_7">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="title">
             <string>Новая комната</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="0" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Название комнаты:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_6"/>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>Добавить комнату</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Длина сообщений</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="spinBox">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>999</number>
               </property>
               <property name="value">
                <number>150</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Пользователи</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="4" column="1">
           <widget class="QComboBox" name="comboBox_3"/>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string>Удалить пользователя</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBox_3">
            <property name="text">
             <string>Создать пользователя</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QGroupBox" name="groupBox_9">
            <property name="title">
             <string>Доступ</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0" colspan="2">
              <widget class="QPushButton" name="pushButton_5">
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Изменить права доступа пользователя</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QGroupBox" name="groupBox_8">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="title">
             <string>Новый пользователь</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_9">
             <item row="1" column="2">
              <widget class="QLineEdit" name="lineEdit_7"/>
             </item>
             <item row="2" column="2">
              <widget class="QLineEdit" name="lineEdit_8">
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>LOGIN</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>PASSWORD</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1" colspan="2">
              <widget class="QPushButton" name="pushButton_3">
               <property name="text">
                <string>Добавить пользователя</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_9">
         <property name="text">
          <string>Удалить схему</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Настройки соединения</string>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>HOST:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>PORT:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>ADMIN LOGIN:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>ADMIN PASSWORD:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_5">
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string>Вход</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="title">
          <string>Схема базы</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox"/>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_8">
            <property name="text">
             <string>Выбрать схему</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Выбрать схему</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>Создать</string>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <item row="2" column="0">
              <widget class="QLabel" name="label_3">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Новая схема</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="lineEdit_2">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="0" colspan="2">
              <widget class="QPushButton" name="pushButton_6">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Создать </string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QCheckBox" name="checkBox">
               <property name="text">
                <string>Создать новую</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Админ панель:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QGroupBox" name="groupBox_10">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Редактирование прав пользователей</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="9" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Добавить права</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0" colspan="2">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Удалить права</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QCheckBox" name="checkBox_4">
         <property name="text">
          <string>Просмотр/отправка сообщений</string>
         </property>
        </widget>
       </item>
       <item row="14" column="2">
        <widget class="QPushButton" name="pushButton_10">
         <property name="text">
          <string>Назначить</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Права пользователя </string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QListView" name="listView"/>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QListView" name="listView_2"/>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Список пользователей</string>
         </property>
        </widget>
       </item>
       <item row="16" column="1" colspan="2">
        <widget class="QComboBox" name="comboBox_5"/>
       </item>
       <item row="17" column="2">
        <widget class="QPushButton" name="pushButton_11">
         <property name="text">
          <string>Назначить</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="3">
        <widget class="QListView" name="listView_3"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>Все комнаты</string>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QCheckBox" name="checkBox_5">
         <property name="text">
          <string>Просмотр/отправка сообщенний</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1" colspan="2">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>Комната</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1" colspan="2">
        <widget class="QComboBox" name="comboBox_4"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>855</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
